### PR TITLE
PTL cleanup_jobs() method should use actual mom host name

### PR DESCRIPTION
**Pull request by birbalsain**
#### Describe Bug or Feature
In PTL, cleanup_jobs() method is not taking the actual hostname of moms when PTL creates multiple moms with create_moms. that time mom naming is mom-0, mom1,mom-2 so on , these are not the actual mom host machine name.

#### Describe Your Change
PTL cleanup_jobs()  method uses actual mom host name.

#### Attach Test and Valgrind Logs/Output

before changes
 
[perf_failed_case.txt](https://github.com/openpbs/openpbs/files/7004060/perf_failed_case.txt)

after changes
mom on server
Description: Tests from given sources on platforms UBUNTU1804, UBUNTU1604, CENTOS8, RHEL8, CENTOS7, RHEL7, SLES12, SLES15, UBUNTU2004
Platforms: UBUNTU1804,UBUNTU1604,CENTOS8,RHEL8,CENTOS7,RHEL7,SLES12,SLES15,UBUNTU2004
Profile: regression
|ID|TOTAL|FAIL|ERROR|TIMEDOUT|SKIP|PASS|
|--|--|--|--|--|--|--|
|8452|3275|2|0|0|0|3273|

no mom on server
Description: Tests from given sources on platforms UBUNTU1804, UBUNTU1604, CENTOS8, RHEL8, CENTOS7, RHEL7, SLES12, SLES15, UBUNTU2004
Platforms: UBUNTU1804,UBUNTU1604,CENTOS8,RHEL8,CENTOS7,RHEL7,SLES12,SLES15,UBUNTU2004
Profile: regression
|ID|TOTAL|FAIL|ERROR|TIMEDOUT|SKIP|PASS|
|--|--|--|--|--|--|--|
|8451|3275|12|8|0|12|3243|

### CODE REVIEW

**Review by visheshh — APPROVED** [review]
LGTM

**Review by visheshh — CHANGES_REQUESTED** [review]
@birbalsain You can use pbsnodes -v host -F json and use key value pairs to fetch mom name and avoid for loop.

File: test/fw/ptl/lib/ptl_server.py [hunk 1]
```diff
@@ -1299,6 +1299,12 @@ def cleanup_jobs(self):
         if len(job_ids) > 100:
             for host, pids in host_pid_map.items():
                 chunks = [pids[i:i + 5000] for i in range(0, len(pids), 5000)]
+                pbsnodes = os.path.join(
+                    self.client_conf['PBS_EXEC'], 'bin', 'pbsnodes')
+                ret = self.du.run_cmd(
+                    self.hostname, [pbsnodes, '-v', host, '-F', 'json'],
+                    logerr=False, level=logging.DEBUG, sudo=True)
+                host = ((ret['out'][6]).split(':'))[1].split('"')[1]
```
Comment (visheshh) [inline test/fw/ptl/lib/ptl_server.py:1307]: join the ret['out'] and use the keys

**Review by visheshh — CHANGES_REQUESTED** [review]

File: test/fw/ptl/lib/ptl_server.py (hunk 1 above)
Comment (visheshh) [reply test/fw/ptl/lib/ptl_server.py:1307]: pbsnodes_json = json.loads('\n'.ret['out']))

**Review by visheshh — APPROVED** [review]
I sign off. Thanks for making changes

**Comment by kjakkali** [comment]
@openpbs/openpbs-maintainers  Please take this forward.

**Review by bayucan — APPROVED** [review]
Looks fine.

**Pull request merged**